docs/debugging-python-lambda-functions.md

```diff
@@ -0,0 +1,92 @@
+# Debugging Python lambda functions
+
+## Install and configure prerequisites
+
+1. Install [Docker](https://www.docker.com/get-started) and start the Docker service.
+2. Install [AWS SAM CLI](https://github.com/awslabs/aws-sam-cli/releases).
+3. Install [Python](https://www.python.org/downloads/). AWS Lambda supports Python 2.7, 3.6, and 3.7.
+4. Install [Visual Studio Code](https://code.visualstudio.com/download).
+5. Install the AWS Toolkit for Visual Studio Code. <!-- TODO: Add marketplace link once the toolkit is published. -->
```

Comment:
Do we intend to publish these instructions at any point between now and launch? If so, link to https://github.com/aws/aws-toolkit-vscode#getting-started . 

Additionally, will we have full integration by launch? If so, and we don't intend to truly publish this before the marketplace link becomes available, this doc loses its value.

Reply:
Should we also have an equivalent doc for Node?

Reply:
Right now, NodeJS is documented in the README. I agree that we should move it to a separate file and make it as equivalent as possible to this one. I'll do that in a separate change.